content/constants.js

```diff
@@ -6,7 +7,12 @@ require("dotenv").config({
 });
 
 const CONTENT_ROOT = process.env.CONTENT_ROOT;
-console.assert(CONTENT_ROOT, "Env var CONTENT_ROOT must be set");
+if (!CONTENT_ROOT) {
+  throw new Error("Env var CONTENT_ROOT must be set");
```

Comment:
this one is kinda new and somewhat unrelated to the issue. But I realize, that `console.assert` was rather useless as it was before. There's absolutely no point carrying on if it's not set.